porch/func/internal/podevaluator.go

```diff
@@ -65,30 +74,109 @@ func NewPodEvaluator(namespace, wrapperServerImage string) (Evaluator, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to create client: %w", err)
 	}
-	return &podEvaluator{
+
+	pe := &podEvaluator{
 		kubeClient:         cl,
 		namespace:          namespace,
 		wrapperServerImage: wrapperServerImage,
-	}, nil
+		gcScanInternal:     time.Duration(interval) * time.Second,
+		podTTL:             time.Duration(ttl) * time.Second,
+		imageEntrypoint:    map[string][]string{},
+		grpcConnectionPool: map[string]*grpc.ClientConn{},
+	}
+	go pe.garbageCollector()
```

Comment:
OK to add TODO for now and implement in a follow-up.
we should have a graceful shutdown for this goroutine.